### ISSUE TITLE
electron-builder@25 crashes if electron-builder.env exists

### ISSUE THREAD

**Issue by Dozorengel** [electron-userland/electron-builder#8451]
* **Electron-Builder Version**: 25.0.0 - 25.0.5

* **Node Version**: 20.17.0

* **Electron Version**: 31.4.0
* **Electron Type (current, beta, nightly)**: current

* **Target**: mac, windows

If there is `electron-builder.env` (even empty), any build command throws an error with the following stacktrace:
```
❯ npx electron-builder
  ⨯ require(...) is not a function  failedTask=build stackTrace=TypeError: require(...) is not a function
    at loadEnv (/Users/dozorengel/projects/glabix-screen-recorder/node_modules/read-config-file/src/main.ts:140:27)
```

If I delete this file, build is working fine. The latest working version is 24.13.3

**Comment by mmaietta** [comment]
I'll take a look at this asap. There's other issues appearing with the updated dependency read-config-file, so I may have to just copy the logic directly into electron-builder for more control of the implementation.

**Comment by mmaietta** [comment]
Quick note: I've extracted read-config-file into electron-builder, updated `dotenv-expand` in this PR https://github.com/electron-userland/electron-builder/pull/8455 and was unable to reproduce the issue, so hopefully that was just a simple fix.

**Comment by emeryao** [comment]
+1 same here

**Comment by mmaietta** [comment]
This should be fixed in electron-builder 25.1 (with app-builder-lib 25.1.1), can you please give that a shot?

**Comment by emeryao** [comment]
`electron-builder@25.1.2` works fine with `.env` file for my project 👍
